cmd/shortener/main.go

```diff
@@ -43,15 +52,18 @@ func main() {
 	router.Use(middleware.RequestLogger)
 	router.Use(middleware.RequestCompressor)
 
-	router.Post("/", apiPageMain)
-	router.Get("/{id}", apiPageByID)
-	router.Post("/api/shorten", apiShorten)
+	shortURLService := NewShortUrlsService(&shorturljsonfile.JSONFileShortURLRepository{})
+
+	router.Post("/", shortURLService.apiPageMain)
+	router.Get("/{id}", shortURLService.apiPageByID)
```

Comment:
Возможно проблема с `chi.URLParam()` сводится к отсутствию закрывающего слэша. Попробуй так:

```suggestion
	router.Get("/{id}/", shortURLService.apiPageByID)
```

Reply:
не помогло(
если добавить слеш маршрутизация на этот адрес вообще перестает работать, на запрос страницы возвращется 404

Reply:
спасибо!